style/components/actors.scss

```diff
@@ -23,3 +23,15 @@
 .actor.npc_hostile {
   @include hostile();
 }
+
+.name {
+  padding: 0 1em;
+}
+
+.init {
+  padding-left: 1em;
+}
+
+input[type=number] {
+  width: 3em;
```

Comment:
In many fonts, this isn't actually the width of 3 chars, but good enough for me here!